src/openlcb/BLEHubPort.hxx

```diff
@@ -60,6 +62,10 @@ public:
     ///
     using SendFunction = std::function<void(const uint8_t *data, size_t len)>;
 
+    /// How big can a single attribute write be? ESP's BLE implementation says
+    /// 600 bytes. We keep some buffer.
+    static constexpr size_t MAX_BYTES_PER_WRITE = 500;
```

Comment:
fyi, ble core specification is 512b. I'd likely drop to ~450b as a default (or move to a config constant)

Reply:
There is apparently also an MTU negotiation happening after the connection gets established. So this likely won't work just as is. Then the negiatiated MTU is not the maximum value that you can transmit, because there is some protocol headers as well.